cluster/addons/fluentd-elasticsearch/README.md

```diff
@@ -8,9 +8,10 @@ is a graphical interface for viewing and querying the logs stored in
 Elasticsearch.
 
 **Note:** this addon should **not** be used as-is in production. This is
-an example and you should treat is as such. Please see at least the
+an example and you should treat it as such. Please see at least the
 [Security](#security) and the [Storage](#storage) sections for more
-information.
+information. Furthermore, the manifests are written with Kubernetes 1.8+ in mind, and
+at least the RBAC manifests must be edited in order to work on earlier versions.
```

Comment:
I would change this to `There's no guarantee it will won with Kubernetes in version 1.7 and earlier`. Who know what broke along the way